packages/reference/src/common/SortableLinkList.tsx

```diff
@@ -17,6 +17,7 @@ const styles = {
   }),
   item: css({
     marginBottom: tokens.spacingM,
+    zIndex: tokens.zIndexModalContent, // setting this to an index above 99 fixes [TOL-823]
```

Comment:
I think instead of giving the ticket number here it would be better to describe the issue briefly (e.g. `fixes dragged item disappearing`). People will see the ticket number in the git blame